pulse_actions/handlers/treeherder_runnable.py

```diff
@@ -3,7 +3,6 @@
 from pulse_actions.utils.misc import whitelisted_users, filter_invalid_builders
 
 from mozci import query_jobs
-from mozci.ci_manager import TaskClusterBuildbotManager
```

Comment:
Please don't remove this.
Do this:
```python
from mozci import TaskClusterBuildbotManager
```
This due to my re-factoring:
https://github.com/mozilla/mozilla_ci_tools/blob/master/mozci/__init__.py#L6